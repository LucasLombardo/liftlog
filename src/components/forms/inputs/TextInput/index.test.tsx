import { TextInput } from "./index";
import { render, screen } from "@testing-library/react-native";

describe("TextInput", () => {
  it("component matches snapshot", () => {
    const handlerFn = () => () => null;
    const props = {
      values: { testInput: "" },
      label: "test input",
      id: "testInput",
      handleBlur: handlerFn,
      handleChange: handlerFn,
    };
    render(<TextInput {...props} />);
    expect(screen).toMatchSnapshot();
  });
});
