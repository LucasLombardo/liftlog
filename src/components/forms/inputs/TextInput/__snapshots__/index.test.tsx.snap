// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput component matches snapshot 1`] = `
<View>
  <Text
    style={
      {
        "fontSize": 16,
      }
    }
  >
    test input
  </Text>
  <TextInput
    accessibilityLabel="test input"
    onBlur={[Function]}
    onChangeText={[Function]}
    style={
      {
        "borderColor": "black",
        "borderWidth": 1,
        "marginBottom": 10,
        "padding": 4,
      }
    }
    value=""
  />
</View>
`;
